nixos/modules/services/misc/ntfy-sh.nix

```diff
@@ -88,17 +79,8 @@ in
           RestrictNamespaces = true;
           RestrictRealtime = true;
           MemoryDenyWriteExecute = true;
-        };
-      };
-
-      users.groups = optionalAttrs (cfg.group == "ntfy-sh") {
-        ntfy-sh = { };
-      };
-
-      users.users = optionalAttrs (cfg.user == "ntfy-sh") {
-        ntfy-sh = {
-          isSystemUser = true;
-          group = cfg.group;
+          # Upstream Requirements
+          LimitNOFILE = 20500;
```

Comment:
Can I ask why you chose this value ? I can see in the ntfy docs that there is a default at 10000 here that we may want to change but 20500 seems arbitrary.

Reply:
I just took it from the docs
They said you might want to increase to 20,000 with 500 for file attachements.
My experience with the file limit is that its a pain to get to, and you dont use more resources increasing the limit if you never actually consume it.
I thought it might prevent a few unpleasant debugging sessions to increase the default, but i dont have a strong opinion at all on this, i dont plant to run it at that scale.

Reply:
Didn't see that there was an example with value 20500 in the docs :) If changing this value has no downside effects for people running a small instance I'm fine with it :)